zebrad/src/commands/start/sync.rs

```diff
@@ -198,97 +197,64 @@ where
             }
         }
 
-        self.request_blocks(download_set.into_iter().collect())
-            .await?;
+        self.prospective_tips
+            .retain(|tip| !download_set.contains(tip));
```

Comment:
:+1: Could be good to add a comment above this line explaining that it prevents us from attempting to download a tip already included behind another tip.